rules/bench-master/rule.md

```diff
@@ -79,7 +79,7 @@ Here are some considerations the Bench Master would keep in mind for every devel
 1. Client needs - Is there any client work that tentatively needs the developer?
 2. Time between now and next client?
 3. Inbox count?
-   * see <[https://www.ssw.com.au/rules/dones-is-your-inbox-a-task-list-only/](/dones-is-your-inbox-a-task-list-only/)>
+   * see [https://www.ssw.com.au/rules/dones-is-your-inbox-a-task-list-only/](/dones-is-your-inbox-a-task-list-only/)
```

Comment:
```suggestion
   * see [https://www.ssw.com.au/rules/dones-is-your-inbox-a-task-list-only/](/rules/dones-is-your-inbox-a-task-list-only/)
```

Reply:
This doesn't need a `/rules` prefix as it's inside the rule body.